packages/nextjs/app/builders/0x972d1be405DcFcD32DD95743934693B673e3F2B0/page.tsx

```diff
@@ -0,0 +1,126 @@
+import Image from "next/image";
+import { NextPage } from "next";
+import { IconType } from "react-icons";
+import { FaEnvelope, FaGithub, FaMapMarkerAlt } from "react-icons/fa";
+
+interface ProfileData {
+  name: string;
+  title: string;
+  bio: string;
+  avatar: string;
+  location: string;
+  email: string;
+  social: {
+    github: string;
+    twitter: string;
+  };
+}
+
+interface StyledIconProps extends React.HTMLAttributes<HTMLSpanElement> {
+  icon: IconType;
+  size?: number;
+  color?: string;
+}
+
+interface SocialLink {
+  href: string;
+  icon: IconType;
+  lightHoverColor: string;
+  darkHoverColor: string;
+}
+
+const StyledIcon: React.FC<StyledIconProps> = ({ icon: Icon, className, size, color, ...props }) => {
+  return (
+    <span className={`inline-block ${className}`} {...props}>
+      <Icon size={size} color={color} />
+    </span>
+  );
+};
+
+// Social links configuration moved outside the component
+const socialLinks: SocialLink[] = [
+  {
+    href: "",
+    icon: FaGithub,
+    lightHoverColor: "hover:text-gray-900",
+    darkHoverColor: "dark:hover:text-gray-200",
+  },
+  {
+    href: "",
+    icon: FaEnvelope,
+    lightHoverColor: "hover:text-red-500",
+    darkHoverColor: "dark:hover:text-red-400",
+  },
+];
+
+const MillburnBuilderPage: NextPage = () => {
+  const profile: ProfileData = {
+    name: "Developer Millburn",
+    title: "Smart Contract Developer",
+    bio: "I'm a blockchain developer, versitle in writing smart contract using Solidity, Hardhat, Foundry, Remix, EthersJs.",
+    avatar: "/blackcat.jpg",
+    location: "Kogi State, Nigeria",
+    email: "millburncrack@gmail.com",
+    social: {
+      github: "https://github.com/MillburnCrackDev",
+      twitter: "https://x.com/Millburn",
+    },
+  };
+
+  // Dynamically populate href values
+  const populatedSocialLinks = socialLinks.map((link, index) => ({
+    ...link,
+    href: [profile.social.github, profile.social.twitter, `mailto:${profile.email}`][index],
+  }));
```

Comment:
There's no need to dinamically populate the href values. They could be hardcoded in the socialLinks array. This would make the code easier to maintain.